src/views/StyledGameFilters.jsx

```diff
@@ -0,0 +1,31 @@
+import styled from "styled-components";
+
+export const StyledGameFilters = styled.ul`
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0;
+`;
+
+export const StyledGameFilter = styled.li`
+  flex: auto;
+  max-width: 30rem;
+  margin: 0 1rem;
+`;
+
+export const StyledGameResultNumber = styled.p`
+  text-align: center;
+`;
+
+export const StyledInput = styled.input`
+  &#game-search {
```

Comment:
That's for now the way I managed to override the default `input` css in the index.css file.

Reply:
Use the same selector to be able to overide it.

Reply:
![image](https://user-images.githubusercontent.com/571533/113519959-0fedbd00-9590-11eb-9624-8fcbee51e476.png)

Reply:
Not sure to understand this comment: do you mean overriding the `:not(...)` selector?